internal/tsetse/tsconfig.json

```diff
@@ -10,6 +10,12 @@
       "es5",
       "es2015.core",
       "es2015.collection"
+    ],
+    "plugins": [
+      // Run `bazel build internal/tsetse:plugin` for this to work.
```

Comment:
what happens if you haven't run it?

how will users install the plugin? will there be some path under `@bazel/typescript` or do we need to publish a separate NPM package for this?

Reply:
If you don't run it, it doesn't work (you get no tsetse squiggles, but everything else works).

Doesn't need to be a separate package; you can just do `@bazel/typescript/path/to/plugin.js`. But they do seem to expect you to publish it as a separate NPM module.

I don't know how NPM and TS interact; do people check in the .js? Or is there some other way to have your JS generated and then made available in NPM?

Reply:
Typically, you assume your users cannot compile your code, so you ship .js/.d.ts to NPM.
With Bazel, we open up the possibility of shipping sources and BUILD files. A bazel-to-bazel dependency can skip shipping .js files (`@bazel/typescript` is a source-only distro)

However, I don't think we want our users to have a broken experience before their first bazel build, so I think this plugin should probably be shipped as executable JS, which means publishing a new package. I can take on that release engineering work after this merges.